tests/plugin.py

```diff
@@ -0,0 +1,31 @@
+"""This file allows to override pytest behavior,
+add new command-line options and test markers."""
+
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
`addoption` is a convention you might not want to let bleed through to the rest of the project.

For example, if we were using a python library that used `camelCase` instead of `snake_case`, if we're going to wrap it, we may as well use `snake_case` in the wrappers name, so that the entire project is more consistently `snake_case`.

so here -> consider `pytest_addoption` -> `pytest_add_option`

Reply:
I 100% agree, however I think it's a pytest specific hook name, so if I change it then it's not going to do its thing (add options on test run), since pytest won't be able to discover it